docs/faq/index.md

```diff
@@ -2,38 +2,38 @@
 
 ## General questions
 
-### What does the GitOps Toolkit mean for Flux?
+### What does Flux v2 mean for Flux?
 
-Flux v1 is a monolithic do-it-all operator; the GitOps Toolkit separates the functionalities into specialized controllers.
+Flux v1 is a monolithic do-it-all operator; Flux v2 separates the functionalities into specialized controllers, collectively called the GitOps Toolkit.
 
-Flux v2 will be a curated configuration of the GitOps Toolkit, which you can install and operate simply using the `flux` command. You can easily pick and choose the functionality you need and extend it to serve your own purposes.
+You can install and operate Flux v2 simply using the `flux` command. You can easily pick and choose the functionality you need and extend it to serve your own purposes.
 
 The timeline we are looking at right now is:
 
 1. Put Flux v1 into maintenance mode (no new features being added; bugfixes and CVEs patched only).
-1. Continue work on [GitOps Toolkit roadmap](https://toolkit.fluxcd.io/roadmap/).
-1. We will provide transition guides for specific user groups, e.g. users of Flux v1 in read-only mode, or of Helm Operator v1, etc. once the functionality is integrated in the GitOps Toolkit and it's deemed "ready".
+1. Continue work on the [Flux v2 roadmap](https://toolkit.fluxcd.io/roadmap/).
+1. We will provide transition guides for specific user groups, e.g. users of Flux v1 in read-only mode, or of Helm Operator v1, etc. once the functionality is integrated into Flux v2 and it's deemed "ready".
 1. Once the use-cases of Flux v1 are covered, we will continue supporting Flux v1 for 6 months. This will be the transition period before it's considered unsupported.
 
 ### Why did you rewrite Flux?
 
-The GitOps Toolkit implements its functionality in individual controllers, which allowed us to address long-standing feature requests much more easily.
+Flux v2 implements its functionality in individual controllers, which allowed us to address long-standing feature requests much more easily.
 
 By basing these controllers on modern Kubernetes tooling (`controller-runtime` libraries), they can be dynamically configured with Kubernetes custom resources either by cluster admins or by other automated tools -- and you get greatly increased observability.
 
-This gave us the opportunity to build the GitOps Toolkit with the top Flux feature requests in mind:
+This gave us the opportunity to build FLux v2 with the top Flux v1 feature requests in mind:
```

Comment:
```suggestion
This gave us the opportunity to build Flux v2 with the top Flux v1 feature requests in mind:
```

Reply:
I knew this one would trip me up, I type that _all the time_

Reply:
I found two more places I'd done that!